src/pages/index.tsx

```diff
@@ -1,9 +1,14 @@
 import { Avatar, Box, Heading, HStack, VStack } from '@chakra-ui/react';
+import { useEffect } from 'react';
 import { trpc } from '../utils/trpc';
 import { NextPageWithLayout } from './_app';
 
 const IndexPage: NextPageWithLayout = () => {
-  const usersQuery = trpc.useQuery(['user.all']);
+  const usersQuery = trpc.proxy.user.all.useQuery();
+
+  useEffect(() => {
+    console.log(usersQuery.data);
+  }, []);
```

Comment:
Codacy has a fix for the issue: [React Hook useEffect has a missing dependency: 'usersQuery.data'. Either include it or remove the dependency array.](https://app.codacy.com/gh/mellson/react-cph-talk/pullRequest?prid=9978673)
```suggestion
  }, [usersQuery.data]);
```